scripts/prepare_sector_network.py

```diff
@@ -2059,23 +2065,25 @@ def add_services(n, costs):
 def add_agriculture(n, costs):
     n.madd(
         "Load",
-        nodes,
+        spatial.nodes,
         suffix=" agriculture electricity",
-        bus=nodes,
+        bus=spatial.nodes,
         carrier="agriculture electricity",
-        p_set=nodal_energy_totals.loc[nodes, "agriculture electricity"] * 1e6 / 8760,
+        p_set=nodal_energy_totals.loc[spatial.nodes, "agriculture electricity"]
+        * 1e6
+        / 8760,
     )
 
     n.madd(
         "Load",
-        nodes,
+        spatial.nodes,
         suffix=" agriculture oil",
         bus=spatial.oil.nodes,
         carrier="agriculture oil",
-        p_set=nodal_energy_totals.loc[nodes, "agriculture oil"] * 1e6 / 8760,
+        p_set=nodal_energy_totals.loc[spatial.nodes, "agriculture oil"] * 1e6 / 8760,

```

Comment:
the same here: does it make sense to write a method for nodal_energy_totals.loc[spatial.nodes, "type"] * 1e6 / 8760?